lib/impl/AzurePresignTvm.js

```diff
@@ -54,7 +62,8 @@ class AzurePresignTvm extends Tvm {
     const commonSasParams = {
       permissions: permissions.toString(),
       expiryTime: expiryTime,
-      blobName: params.blobName
+      blobName: params.blobName,
+      identifier: identifier
```

Comment:
The name `identifier` does not provide sufficient information, what about `policyId` ?